MAUI/Masked-Entry/Basic-Features.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Basic Features in .NET MAUI Masked Entry control | Syncfusion
+description:  Learn here all about Basic Features support in Syncfusion .NET MAUI Masked Entry (SfMaskedEntry) control and more.
+platform: maui
+control: SfMaskedEntry
+documentation: ug
+---
+
+# Basic Features in .NET MAUI Masked Entry (SfMaskedEntry)
+
+## Setting ClearButtonVisibility
+
+ClearButtonVisibility, of type ClearButtonVisibility, controls whether a clear button is displayed, which enables the user to clear the text. The default value of this property is `ClearButtonVisibility.Never`.
```

Comment:
The ClearButtonVisibility of type ClearButtonVisibility controls whether a clear button is displayed, which enables the user to clear the text. The default value of this property is `ClearButtonVisibility.Never`.

Reply:
Addressed the changes.